app/src/js/utils/table-config/reconciliation-reports.js

```diff
@@ -252,3 +253,38 @@ export const tableColumnsGnf = [
     width: 50,
   },
 ];
+
+export const tableColumnsBackupAndRecovery = [
+  {
+    Header: 'Collection ID',
+    accessor: 'collectionId',
+    // eslint-disable-next-line react/prop-types
+    Cell: ({ cell: { value } }) => <Link to={(location) => ({
+      pathname: collectionHrefFromId(value), search: getPersistentQueryParams(location)
+    })}>{value}</Link>,
+    width: 125,
+  },
+  {
+    Header: 'Granule ID',
+    accessor: 'granuleId',
+    width: 200,
+  },
+  {
+    Header: 'Provider',
+    accessor: 'provider'
+  },
+  {
+    Header: 'Conflict Type',
+    id: 'conflictType'
+  },
+  // {
```

Comment:
Are we keeping this Conflict Details section commented for a reason?

Reply:
The display isn't working on the PR, and my new PR for the display ticket will reorganize the fields, and add more information.